{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-11T12:10:00.000+0000",
      "latitude": 50.90126572064888,
      "estimatedRestoration": "2022-08-20T01:50:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY1106'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY1106",
      "reference": "JY1106",
      "maxLongitude": -1.5082,
      "engineerOnSiteTime": "2022-08-11T13:15:00.000+0000",
      "minLatitude": 50.8991,
      "name": "SO40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO40 7EE",
        "SO40 7EF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5142,
              50.9029
            ],
            [
              -1.5082,
              50.9029
            ],
            [
              -1.5082,
              50.8991
            ],
            [
              -1.5142,
              50.8991
            ],
            [
              -1.5142,
              50.9029
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "45ac421b-9e60-4d0a-a008-c59ba3c969c5",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.5142,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.9029,
      "longitude": -1.512038514415299,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T00:52:00.000+0000",
      "latitude": 50.82928558768313,
      "estimatedRestoration": "2022-08-20T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7255'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7255",
      "reference": "JY7255",
      "maxLongitude": -0.7806,
      "engineerOnSiteTime": "2022-08-20T01:30:00.000+0000",
      "minLatitude": 50.8262,
      "name": "PO19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO19 8DS",
        "PO19 8DT",
        "PO19 8QH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7857,
              50.8311
            ],
            [
              -0.7806,
              50.8311
            ],
            [
              -0.7806,
              50.8262
            ],
            [
              -0.7857,
              50.8262
            ],
            [
              -0.7857,
              50.8311
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "e25e53a7-cff2-4058-9dbb-52867df98736",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.7857,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8311,
      "longitude": -0.7836811017837627,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T01:30:00.000+0000",
      "latitude": 51.42684437056087,
      "estimatedRestoration": "2022-08-20T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7259'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7259",
      "reference": "JY7259",
      "maxLongitude": -2.1915,
      "engineerOnSiteTime": "2022-08-20T02:00:00.000+0000",
      "minLatitude": 51.4259,
      "name": "SN13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN13 9NL",
        "SN13 9NQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1942,
              51.4277
            ],
            [
              -2.1915,
              51.4277
            ],
            [
              -2.1915,
              51.4259
            ],
            [
              -2.1942,
              51.4259
            ],
            [
              -2.1942,
              51.4277
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "815399b6-5453-4e2d-a7a4-4912a986c001",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.1942,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4277,
      "longitude": -2.1929156523192295,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T07:26:00.000+0000",
      "latitude": 50.72181329084829,
      "estimatedRestoration": "2022-08-20T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7344'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7344",
      "reference": "JY7344",
      "maxLongitude": -2.0118,
      "engineerOnSiteTime": "2022-08-20T08:00:00.000+0000",
      "minLatitude": 50.7209,
      "name": "BH15 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH15 4HU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0148,
              50.7229
            ],
            [
              -2.0118,
              50.7229
            ],
            [
              -2.0118,
              50.7209
            ],
            [
              -2.0148,
              50.7209
            ],
            [
              -2.0148,
              50.7229
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-08-20T08:15:00.000+0000",
      "UUID": "aed1be5c-45a9-4120-94a1-bb0ff698ae89",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.0148,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7229,
      "longitude": -2.0133068285873494,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T08:30:00.000+0000",
      "latitude": 50.74138206936828,
      "estimatedRestoration": "2022-08-20T19:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7376'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7376",
      "reference": "JY7376",
      "maxLongitude": -1.9972,
      "engineerOnSiteTime": "2022-08-20T08:55:00.000+0000",
      "minLatitude": 50.7401,
      "name": "BH17 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH17 7DN",
        "BH17 7DW",
        "BH17 7EG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0044,
              50.743
            ],
            [
              -1.9972,
              50.743
            ],
            [
              -1.9972,
              50.7401
            ],
            [
              -2.0044,
              50.7401
            ],
            [
              -2.0044,
              50.743
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-08-20T09:00:00.000+0000",
      "UUID": "746b92ba-baa5-493b-9068-4a606ce46703",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.0044,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.743,
      "longitude": -2.0006709418121855,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T09:44:00.000+0000",
      "latitude": 50.872862874612935,
      "estimatedRestoration": "2022-08-20T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7381'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7381",
      "reference": "JY7381",
      "maxLongitude": -1.2885,
      "engineerOnSiteTime": "2022-08-20T09:21:00.000+0000",
      "minLatitude": 50.8687,
      "name": "SO31 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO31 7AD",
        "SO31 7AE",
        "SO31 7AF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.297,
              50.8763
            ],
            [
              -1.2885,
              50.8763
            ],
            [
              -1.2885,
              50.8687
            ],
            [
              -1.297,
              50.8687
            ],
            [
              -1.297,
              50.8763
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-08-20T09:30:00.000+0000",
      "UUID": "b85eda8a-1441-4e31-9cbd-ad18ec2dfacd",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.297,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.8763,
      "longitude": -1.2932266579205094,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T10:15:00.000+0000",
      "latitude": 50.94748255539674,
      "estimatedRestoration": "2022-08-20T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7422'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7422",
      "reference": "JY7422",
      "name": "SP6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP6 3PL"
      ],
      "location": {
        "coordinates": [
          -1.806613228822429,
          50.94748255539674
        ],
        "type": "Point"
      },
      "UUID": "1d7e0994-3304-4486-be43-6663afe0edca",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.806613228822429,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T14:31:00.000+0000",
      "latitude": 51.03495511864222,
      "estimatedRestoration": "2022-08-20T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7503'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7503",
      "reference": "JY7503",
      "maxLongitude": -2.2708,
      "minLatitude": 51.034,
      "name": "SP8 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP8 4QA",
        "SP8 4QQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2752,
              51.0361
            ],
            [
              -2.2708,
              51.0361
            ],
            [
              -2.2708,
              51.034
            ],
            [
              -2.2752,
              51.034
            ],
            [
              -2.2752,
              51.0361
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "c997070c-ee20-4a2b-8996-4fc73eddf02d",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.2752,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.0361,
      "longitude": -2.2720874747505224,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T13:54:00.000+0000",
      "latitude": 51.619322221172666,
      "estimatedRestoration": "2022-08-20T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7489'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7489",
      "reference": "JY7489",
      "maxLongitude": -0.7328,
      "minLatitude": 51.6169,
      "name": "HP11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP11 1DT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.738,
              51.6199
            ],
            [
              -0.7328,
              51.6199
            ],
            [
              -0.7328,
              51.6169
            ],
            [
              -0.738,
              51.6169
            ],
            [
              -0.738,
              51.6199
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-08-20T15:00:00.000+0000",
      "UUID": "53bebae7-971b-4cef-9fba-519e471667fb",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.738,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.6199,
      "longitude": -0.7349893903486229,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T15:05:00.000+0000",
      "latitude": 51.2811588907573,
      "estimatedRestoration": "2022-08-20T16:05:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7521'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7521",
      "reference": "JY7521",
      "maxLongitude": -0.8254,
      "minLatitude": 51.2697,
      "name": "GU51 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU13",
        "GU51",
        "GU51 1AD",
        "GU51 1DD",
        "GU51 1DF",
        "GU51 1DH",
        "GU51 1EA",
        "GU51 1EB",
        "GU51 3AA",
        "GU51 3AB",
        "GU51 3AD",
        "GU51 3AF",
        "GU51 3AG",
        "GU51 3AH",
        "GU51 3AJ",
        "GU51 3AL",
        "GU51 3AN",
        "GU51 3AQ",
        "GU51 3AS",
        "GU51 3AU",
        "GU51 3AW",
        "GU51 3BD",
        "GU51 3BE",
        "GU51 3BF",
        "GU51 3BG",
        "GU51 3BH",
        "GU51 3BJ",
        "GU51 3BS",
        "GU51 3BU",
        "GU51 3DH",
        "GU51 3DJ",
        "GU51 3DL",
        "GU51 3FF",
        "GU51 3FL",
        "GU51 3GE",
        "GU51 3GG",
        "GU51 3HA",
        "GU51 3LE",
        "GU51 3LJ",
        "GU51 3LN",
        "GU51 3LP",
        "GU51 3LR",
        "GU51 3LS",
        "GU51 3LU",
        "GU51 3LW",
        "GU51 3LX",
        "GU51 3LY",
        "GU51 3NA",
        "GU51 3NG",
        "GU51 3NJ",
        "GU51 3PE",
        "GU51 3PF",
        "GU51 3PH",
        "GU51 3PJ",
        "GU51 3QB",
        "GU51 3QD",
        "GU51 3QE",
        "GU51 3QF",
        "GU51 3QT",
        "GU51 3QU",
        "GU51 3QX",
        "GU51 3RA",
        "GU51 3RB",
        "GU51 3RD",
        "GU51 3RE",
        "GU51 3RF",
        "GU51 3RG",
        "GU51 3RL",
        "GU51 3RN",
        "GU51 3RP",
        "GU51 3RQ",
        "GU51 3RR",
        "GU51 3RS",
        "GU51 3RT",
        "GU51 3RU",
        "GU51 3RX",
        "GU51 3RY",
        "GU51 3RZ",
        "GU51 3SD",
        "GU51 3SG",
        "GU51 3SH",
        "GU51 3ST",
        "GU51 3SU",
        "GU51 3SX",
        "GU51 3TH",
        "GU51 3TZ",
        "GU51 3UD",
        "GU51 3US",
        "GU51 3XS",
        "GU51 3XU",
        "GU51 3XX",
        "GU51 3XZ",
        "GU51 3YR",
        "GU51 4NA",
        "GU51 4PA",
        "GU51 4PP",
        "GU51 4PR",
        "GU51 4PS",
        "GU51 4PT",
        "GU51 4PU",
        "GU51 4PW",
        "GU51 4PX",
        "GU51 4PY",
        "GU51 4PZ",
        "GU51 4QA",
        "GU51 4QG",
        "GU51 5DT",
        "GU52 7JZ",
        "GU52 7LB",
        "GU52 7LJ",
        "GU52 7LP",
        "GU52 7LR",
        "GU52 7LS",
        "GU52 7LT",
        "GU52 7LU",
        "GU52 7LX",
        "GU52 7LY",
        "GU52 7QP",
        "GU52 7RH",
        "GU52 7RX",
        "GU52 7RZ",
        "GU52 7SA",
        "GU52 7SB",
        "GU52 7SD",
        "GU52 7SE",
        "GU52 7SF",
        "GU52 7SG",
        "GU52 7SH",
        "GU52 7SJ",
        "GU52 7SL",
        "GU52 7SN",
        "GU52 7SP",
        "GU52 7SQ",
        "GU52 7SR",
        "GU52 7SS",
        "GU52 7ST",
        "GU52 7SW",
        "GU52 7SY",
        "GU52 7SZ",
        "GU52 7TA",
        "GU52 7TB",
        "GU52 7TF",
        "GU52 7TH",
        "GU52 7UA",
        "GU52 7UF",
        "GU52 7YD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8643,
              51.2942
            ],
            [
              -0.8254,
              51.2942
            ],
            [
              -0.8254,
              51.2697
            ],
            [
              -0.8643,
              51.2697
            ],
            [
              -0.8643,
              51.2942
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "b9459e1f-2f77-4672-bde3-353a38b87c1d",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.8643,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.2942,
      "longitude": -0.8382297111950708,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T13:53:00.000+0000",
      "latitude": 57.44450550384239,
      "estimatedRestoration": "2022-08-20T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7488'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7488",
      "reference": "JY7488",
      "maxLongitude": -2.0644,
      "minLatitude": 57.4244,
      "name": "AB41 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB41 7JR",
        "AB41 7TP",
        "AB41 7TQ",
        "AB41 7TR",
        "AB41 8TH",
        "AB41 8TJ",
        "AB41 8TN",
        "AB41 8XN",
        "AB41 8XY",
        "AB41 8YA",
        "AB41 8YB",
        "AB41 8YD",
        "AB41 8YE",
        "AB41 8YG",
        "AB41 8YH",
        "AB41 8YJ",
        "AB41 8YL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2003,
              57.4672
            ],
            [
              -2.0644,
              57.4672
            ],
            [
              -2.0644,
              57.4244
            ],
            [
              -2.2003,
              57.4244
            ],
            [
              -2.2003,
              57.4672
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "7ed9f4bb-2a6a-4c34-a7b7-e71ce405082a",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.2003,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 57.4672,
      "longitude": -2.143995322459046,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-08-20T14:37:00.000+0000",
      "latitude": 55.990127097967196,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JY7502'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JY7502",
      "reference": "JY7502",
      "maxLongitude": -5.6125,
      "minLatitude": 55.9289,
      "name": "PA31 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PA31",
        "PA31 8PJ",
        "PA31 8PN",
        "PA31 8PP",
        "PA31 8PQ",
        "PA31 8PW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.7469,
              56.0457
            ],
            [
              -5.6125,
              56.0457
            ],
            [
              -5.6125,
              55.9289
            ],
            [
              -5.7469,
              55.9289
            ],
            [
              -5.7469,
              56.0457
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-08-20T15:15:00.000+0000",
      "UUID": "142b8d13-04d8-4535-9c15-bb1ea323e24a",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -5.7469,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 56.0457,
      "longitude": -5.663585052250764,
      "resolved": false
    }
  ]
}
